lnxrouter

```diff
@@ -1715,8 +1737,15 @@ write_hostapd_conf() {
 
     if [[ $MAC_FILTER -eq 1 ]]; then
         cat <<- EOF >> "$CONFDIR/hostapd.conf"
-			macaddr_acl=${MAC_FILTER}
-			accept_mac_file=${MAC_FILTER_ACCEPT}
+			macaddr_acl=1
+			accept_mac_file=${MAC_FILTER_FILE}
+		EOF
+    fi
+
+    if [[ $MAC_FILTER -eq 0 ]]; then
+        cat <<- EOF >> "$CONFDIR/hostapd.conf"
+			macaddr_acl=0
+			deny_mac_file=${MAC_FILTER_FILE}
```

Comment:
I'm thinking ...
maybe we can make the script option like this:
```
--mac-filter-accept <accept_file>
--mac-filter-deny <deny_file>
```
then variable `MAC_FILTER_FILE` and option `--mac-filter-file` would be unnecessary .
What do you think?